fix: Backend startup with BASE_HREF set

JVMerkle opened the pull request:
Backend does not need BASE_HREF patching and crashes if a BASE_HREF is set in the entrypoint.sh.

TomBursch commented:
This is needed by the all in one container. A check if the folder/file exists should be enough to handle the backend only case.

Commit:
fix: Backend startup with BASE_HREF set

Backend does not need BASE_HREF patching and crashes if a BASE_HREF is set in the
entrypoint.sh.

Fixes #632

JVMerkle commented:
Done, added an extra file check.
